{"name": "test_6_get_unknown_by_id_not_found", "status": "passed", "start": 1709527086933, "stop": 1709527089472, "uuid": "1ae84966-2687-4ac0-8e62-ff77b20323be", "historyId": "f6ea42f65744e587c0e6843e8ce3d238", "testCaseId": "f6ea42f65744e587c0e6843e8ce3d238", "fullName": "Tests.test_api.TestLogin#test_6_get_unknown_by_id_not_found", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('api_fixture')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-UEVTOM8"}, {"name": "thread", "value": "23820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_api"}]}